### PR TITLE
Block Patterns: Update to apply deprecations

### PR DESCRIPTION
**Pull request by ockham**
#### Changes proposed in this Pull Request

**Don't merge before GB 8.9 is running in production on WP.com.** While merging this PR won't immediately deploy the changes to WP.com, anyone else publishing a new version of the Editing Toolkit Plugin would :scream:

Update block patterns so deprecations/migrations won't be run. 

_(We've seen perf issues with block patterns in GB 8.9 that are potentially due to this. **The problem is particularly present in Firefox, where pattern previews seem to basically stop loading.**)_

I'm creating this PR by basically looking at the updated block markup from the browser console when selecting the block patterns tab in the inserter on an Edge site (running GB 8.9).
image

Since at least a few of these seem to be due to a `core/paragraph` [deprecation](https://github.com/WordPress/gutenberg/blob/458277c376d9b0402ea04034a055c98d7e18625e/packages/block-library/src/paragraph/deprecated.js) that [makes a number of block attributes implicit](https://github.com/WordPress/gutenberg/blob/458277c376d9b0402ea04034a055c98d7e18625e/packages/block-library/src/paragraph/deprecated.js#L78-L82), it's possible that updating the block content alone isn't sufficient, but that we need to also remove the now-implicit attributes, e.g. https://github.com/Automattic/wp-calypso/blob/c1253ea6433191bb24104d38cd8ad8ccdef8e26a/apps/editing-toolkit/editing-toolkit-plugin/block-patterns/patterns/call-to-action-02.php#L25

However, simply copying over the updated markup didn't do the trick; in many cases, I had to change the block attributes instead. This required looking at each affected block type's deprecations/migrations.

#### Testing instructions

- Make sure your sandboxed site has the `gutenberg-edge` sticker.
- Apply the generated diff (D49160-code) to your sandbox.
- Start a new post.
- Open the browser console.
- Open the block inserter, and click the 'Patterns' tab
- There should be no block deprecations happening in the browser console. Prior to this PR, there were 11 of them.

Furthermore, try in Firefox, and see if block pattern thumbnails load (doesn't seem like :slightly_frowning_face: )

#### TODO

In the long run, we need an automated way of running deprecations on the patterns (while preserving dynamic content such as `sprintf` interpolation placeholders (`$1%s` etc) or concatenations (`. esc_html( 'foobar' ) .`)). Wondering if the new Blocks API's i18n technique (that I'm unfamiliar with) allows for that?

cc/ @Automattic/good-mountain @ianstewart

### CODE REVIEW

File: apps/editing-toolkit/editing-toolkit-plugin/block-patterns/patterns/call-to-action-02.php [hunk 1]
```diff
@@ -14,16 +14,16 @@
          <!-- wp:jetpack/layout-grid-column -->
          <div class="wp-block-jetpack-layout-grid-column wp-block-jetpack-layout-grid__padding-none">
             <!-- wp:paragraph {"align":"center","customTextColor":"#ffffff","style":{"typography":{"fontSize":80}},"className":"margin-bottom-none"} -->
-            <p style="color:#ffffff;font-size:80px" class="has-text-color has-text-align-center margin-bottom-none"><strong>' . esc_html__( 'Get it delivered', 'full-site-editing' ) . '</strong></p>
+            <p class="has-text-align-center" style="font-size:80px"><p style="color:#ffffff;font-size:80px" class="has-text-color has-text-align-center margin-bottom-none"><strong>' . esc_html__( 'Get it delivered', 'full-site-editing' ) . '</strong></p></p>
```
Comment (lsl) [inline apps/editing-toolkit/editing-toolkit-plugin/block-patterns/patterns/call-to-action-02.php:17]: Is this nested `<p>` intentional? That will cause problems going through supports code probably.

Comment (ockham) [reply apps/editing-toolkit/editing-toolkit-plugin/block-patterns/patterns/call-to-action-02.php:17]: Yeah, this one's a bit funky. I flagged them in Slack (p1599313110366800-slack-C015AL3QL7M); my hunch is that the paragraph block deprecation isn't working quite right here :slightly_frowning_face: 

image (11)

Comment (ockham) [reply apps/editing-toolkit/editing-toolkit-plugin/block-patterns/patterns/call-to-action-02.php:17]: I'll try a manual fix for now.

**Comment by sirreal** [comment]
👋  @Automattic/view-engineering Can y'all take this over? I know you're working to land new patterns so you may have already addressed these issues.

When the block patterns are rendered in the editor, they show a lot of block deprecations and migrations every time they're rendered. The serialized blocks in the patterns should all be updated to their latest versions to save running those migrations repeatedly.

**Comment by sirreal** [comment]
@apeatling Any ideas on when this work will move forward?

**Comment by sirreal** [comment]
The issue remains, but these changes are no longer relevant. Created a dedicated issue: https://github.com/Automattic/wp-calypso/issues/49404

**Pull request closed without merging**